### ISSUE TITLE
Unexpected layout

### ISSUE THREAD

**Issue by cpetrov** [eclipsesource/tabris-js#955]
Layouting [camera.js](https://github.com/eclipsesource/tabris-maps/blob/examples/example/www/pages/camera.js) from [eclipsesource/tabris-maps](https://github.com/eclipsesource/tabris-maps) results into the [`cameraTextView`](https://github.com/eclipsesource/tabris-maps/blob/examples/example/www/pages/camera.js#L43) not taking up expected space in the UI.

Initially, the text view is not shown. Changing orientation leads to the text being shown as expected. 

| Result |  Expected Result |
| --- | --- |
| <img width="487" alt="screen shot 2016-08-26 at 12 38 05" src="https://cloud.githubusercontent.com/assets/2341769/18010451/d825b0c0-6bb0-11e6-83c7-af43d4d067c0.png"> | <img width="487" alt="screen shot 2016-08-26 at 12 38 54" src="https://cloud.githubusercontent.com/assets/2341769/18010476/ee3d2b54-6bb0-11e6-8be8-37951e5719a8.png"> |

**Comment by patrykmol** [comment]
@cpetrov Can you retest this with 2.0?

**Comment by cpetrov** [comment]
@patrykmol :+1: Yes. It is still reproducible with 2.0.0-nightly (tabris-js and iOS platform built on 17.05.2017)

**Comment by karolszafranski** [comment]
I think that this is not layout issue. `cameraTextView` has empty string in `text` property, that's why it's not visible, and it's intrinsic content height is equal to 0.

I did debug a little JS code and obviously this callback is not called:
```js
    this.on('change:camera', function() {
      updateCameraTextView('Camera changed programmatic');
    })
```

However it's not called in JS, but it is called by iOS client. When I set a breakpoint in `Map.js` [line 21](https://github.com/eclipsesource/tabris-plugin-maps/blob/master/www/Map.js#L21) it is triggered after map appear on the screen.

@cpetrov @ralfstx I ask you for help with further investigation of JS.
